src/selections/selection-test.js

```diff
@@ -73,13 +73,15 @@ let linkSelectionTest = function(payload) {
 	);
 };
 
-let imageSelectionTest = function(payload) {
-	let selectionData = payload.data.selectionData;
+const imageSelectionTest = function(payload) {
+	const selectionData = payload.data.selectionData;
+	const element = selectionData.element;
+	const hasImage = !!element && !!element.findOne('img');
+	const isImage = !!element && element.getName() === 'img';
 
 	return !!(
-		selectionData.element &&
-		selectionData.element.getName() === 'img' &&
-		!selectionData.element.isReadOnly()
+		element &&
+		(hasImage || isImage)
```

Comment:
Tiny nit here: on lines 83 and 84 you're relying on the truthy/falsiness of the expressions, but everywhere else you are coercing to booleans with `!!`. Personally I think The JS Way™ is probably to rely on mere truthy/falsiness, until you get to the function boundary, where you return an actual boolean.